scripts/zones/Sea_Serpent_Grotto/mobs/Charybdis.lua

```diff
@@ -69,7 +69,8 @@ entity.phList =
 }
 
 entity.onMobInitialize = function(mob)
-    mob:setMobMod(xi.mobMod.MULTI_HIT, 5)
+    mob:setMobMod(xi.mobMod.BASE_DAMAGE_MULTIPLIER, 150)
+    mob:setMobMod(xi.mobMod.MULTI_HIT, 5) -- TODO : This is not correct. Will need to be updated to proper multi-hit handling later.
```

Comment:
can you change this to 8 while you are here

it's much closer than 5 is
```suggestion
    mob:setMobMod(xi.mobMod.MULTI_HIT, 8) -- TODO : This is not correct. Will need to be updated to proper multi-hit handling later.
```

Reply:
<img width="901" height="89" alt="Image" src="https://github.com/user-attachments/assets/804c26f0-21d6-477e-9e28-9fe9893ba734" />

Reply:
Done!